fastcrypto/src/vrf.rs

```diff
@@ -239,8 +239,12 @@ pub mod ecvrf {
             let h = public_key.ecvrf_encode_to_curve(alpha_string);
 
             let challenge = RistrettoScalar::from(&self.c);
-            let u = RistrettoPoint::generator() * self.s - public_key.0 * challenge;
-            let v = h * self.s - self.gamma * challenge;
+            let u = RistrettoPoint::multiscalar_mul(
+                [self.s, -challenge],
+                [RistrettoPoint::generator(), public_key.0],
+            )
+            .unwrap();
+            let v = RistrettoPoint::multiscalar_mul([self.s, -challenge], [h, self.gamma]).unwrap();
```

Comment:
As we're planning to remove or justify any unwraps across our codebase (unless they can absolutely never happen) we generally prefer Result<> to avoid panics, no?

Reply:
That makes sense. I've added ?'s to make sure any errors are propagated, but since an error is only returned if the sizes of the iterables do not match, it will never happen here anyway.